sound/soc/sof/intel/hda-ctrl.c

```diff
@@ -158,16 +158,18 @@ void hda_dsp_ctrl_misc_clock_gating(struct snd_sof_dev *sdev, bool enable)
  */
 int hda_dsp_ctrl_clock_power_gating(struct snd_sof_dev *sdev, bool enable)
 {
+	struct sof_intel_hda_dev *hda = sdev->pdata->hw_pdata;
```

Comment:
not clear about the last sentence of the commit message

"
This should be checked when trying
to enable the DMI L1 during the post_fw_run op.
"

is this done in this PR, or is this describing something more of a TODO action?